### ISSUE TITLE
Inappropriate likes

### ISSUE THREAD

**Issue by jufelka** [InstaPy/InstaPy#627]
Hi. 
And I have such settings in my script
```
session.set_upper_follower_count(limit=2000)
session.like_by_tags(['x1', 'x2', 'x3', 'x4'], amount=50)
```
So if account have more than 2000 followers I can't like it. But it writes as I liked it. Here is log file:
Liked: 124
Inappropriate: 76
These 76 inappropriate likes are from accounts with more than 2000 followers. How can I ignore it and make 200 likes in one session?

**Comment by timgrossmann** [comment]
@xxxZedxxx As of now, you can't

**Comment by TR0X** [comment]
Would be greate if you add a feature which actually counts the liked content. At the moment, it's a little bit hard to plan the overall daily likes 'cause of that.

**Comment by timgrossmann** [comment]
@TR0X I don't really know what the point of having exact amounts is? 
Don't you think it's way less conspicuous if you don't like exactly 200 posts of this one tag per day?

**Comment by TR0X** [comment]
Let me rephrase it: It would be great to have the possibiliy to set a daily max. limit of likes/follows to be sure to not cross the limit line. As you've already pointed out, an exact limit is a little bit suspicious. I would suggest to generate a daily limit within a manually set min. and a max. limit.
But thats just my 2 cents, it's a great tool anyway!

**Comment by timgrossmann** [comment]
@TR0X Now I get it 😉
